[DOCS] Replaces outdated integration guides with redirects to versioned documentation (#8173)
great-expectations/great_expectations: docs/docusaurus/docs/integrations/integration_zenml.md

sidebar_custom_props: { icon: 'img/integrations/zenml_icon.png' }
---

> ZenML is an extensible, open-source framework for creating production-ready, portable machine learning (ML) pipelines.  Integrating Great Expectations (GX) with ZenML helps data scientists and ML engineers to make data profiling and validation an integral part of their

Reviewer: ```suggestion
ZenML is an extensible, open-source framework for creating production-ready, portable machine learning (ML) pipelines.  Integrating Great Expectations (GX) with ZenML can help make data profiling and validation an integral part of your production ML toolset and workflows.
```

> production ML toolset and workflows.

Reviewer: ```suggestion

```
